strace_process_tree.py

```diff
@@ -17,8 +17,25 @@
 import re
 import string
 import sys
-from collections import defaultdict, namedtuple
-from functools import partial
+from abc import ABC
+from collections import defaultdict
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    NamedTuple,
+    Optional,
+    Set,
+    Tuple,
+)
+
+
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
```

Comment:
Ah, CI claims this line is not tested (on 3.8).

I usually do

```python
try:
    from typing import Literal  # Python >= 3.8
except ImportError:
    from typing_extensions import Literal
```

and mention `except ImportError` in `.coveragerc`'s `exclude_lines`.

Reply:
I tried that at first but mypy does not like that: https://mypy.readthedocs.io/en/stable/runtime_troubles.html#using-new-additions-to-the-typing-module

Reply:
Curious: how does mypy complain about the try/except version?

Reply:
FWIW I'm not married to the try/except suggestion.  I low-key like it because it requires only one pragma to satisfy coverage.py, as the `import` statement inside the try block is always executed anyway.  The sys.version check would `# pragma: nocover` comments on both `import` statements.

Reply:
```
strace_process_tree.py:37: error: Incompatible import of "Literal" (imported name has type "typing_extensions._SpecialForm", local name has type "<typing special form>")  [assignment]
```